README.md

```diff
@@ -15,6 +14,73 @@ If you need assistance or want to help, reach out to the [`bevy_cli` working gro
 [`bevy_cli` working group channel]: https://discord.com/channels/691052431525675048/1278871953721262090
 [Bevy Discord]: https://discord.gg/bevy
 
+## Installation
+
+At this point, the CLI is not published as a package yet and needs to be installed via git:
+
+```cli
+cargo install --git https://github.com/TheBevyFlock/bevy_cli bevy_cli
```

Comment:
```suggestion
cargo install --git https://github.com/TheBevyFlock/bevy_cli --locked bevy_cli
```

I would prefer to use `--locked` in the install command so users installing `bevy_cli` will have the same dependency versions as contributors developing `bevy_cli`. (This avoids subtle bugs that cannot be reproduced easily.)